### ISSUE TITLE
Lambda problems in summarizer process

### ISSUE THREAD

**Issue by Jrecos** [pandas-profiling/pandas-profiling#664]
Hi, 

I'm currently using Pandas-profiling to get information from a parquet file.
My dataframe contains 140 columns.

When I try to generate the profiling report, I'm getting this error:

`TypeError: <lambda>() argument after * must be an iterable, not bool`

I tried to change the column position but the error persists.

My Code is:

```
df = pd.read_parquet("ConsultationForm.parquet", columns= sorted(columnNames),engine="pyarrow")

df = df.fillna(value=np.nan)

df.head()

profile = ProfileReport(df, title="Pandas Profiling Report", explorative=True, minimal=True)
profile.to_file('ConsultationForm_profiling3.html')
```

I'm doing something wrong? 

Thanks

**Comment by sbrugman** [comment]
Will look into this. What version are you on? And could you please provide a sample of your dataset / minimal version of the data to reproduce (may be anonymized)?

**Comment by Jrecos** [comment]
I'm using the last version.
 How can I provide the dataset?

**Comment by sbrugman** [comment]
You can just drag it into this comment field and github should upload/share it for you

**Comment by Jrecos** [comment]
[ds.zip](https://github.com/pandas-profiling/pandas-profiling/files/5788644/ds.zip)
Done :)

**Comment by sbrugman** [comment]
Good catch, it's a special case when one or more columns are fully empty.
Until I've patched it, you can use this workaround (dropping empty columns):

```
df = df.dropna(how='all', axis=1)
```

**Comment by Jrecos** [comment]
Awsome, Works!

**Comment by sbrugman** [comment]
@Jrecos I've just released v2.10.1 that should resolve your issue!

_Want me to continue to support this package? Please consider [sponsoring](https://github.com/sponsors/sbrugman)_
